.github/workflows/postman-collection-runner.yml

```
@@ -97,23 +97,37 @@ jobs:
         env:
           DATABASE_URL: postgres://db_user:db_pass@localhost:5432/hyperswitch_db
         run: diesel migration run
-
+      
+      - name: Build project
+        if: ${{ ((github.event_name == 'pull_request') && (github.event.pull_request.head.repo.full_name == github.event.pull_request.base.repo.full_name)) || (github.event_name == 'merge_group')}}
+        run: cargo build
```

We can be specific here about the package and binary required to be built.

```suggestion
        run: cargo build --package router --bin router
```